catalog-rest-service/src/main/java/org/openmetadata/catalog/security/CatalogAuthorizer.java

```diff
@@ -27,6 +28,9 @@
    */
   boolean hasPermissions(AuthenticationContext ctx, EntityReference entityReference);
 
+  /** Check if the authenticated user is authorized to perform the operation on the target entity. */
```

Comment:
probably good to rename this as Authorizer itself.

Reply:
Agree, splitting this renaming change to another PR (since it touches a lot of files and hides the PolicyEvaluator implementation)

Reply:
#1966